### DISCUSSION TITLE
When do organicmaps data get synced with osm data?

### DISCUSSION THREAD

**Discussion by csidirop** [organicmaps/organicmaps#944] [Q&A] [answered]
Is there any period in witch the map data is pulled from osm? Just asking out of curiosity.

**Comment by biodranik** [comment, marked as answer]
Now it's roughly once per month, we aim to once per week. Check Settings => About for the exact date.

**Reply by csidirop** [reply]
Oh, its the "Dataversion" date?

**Reply by biodranik** [reply]
Yes.
